### DISCUSSION TITLE
NA28 support?

### DISCUSSION THREAD

**Discussion by MC1171611** [mdbergmann/Eloquent#81] [Ideas]
Hi Manfred,

Eloquent is my "daily driver" Bible app and I'm wondering if you've ever implemented support for NA28 unlock keys from DBG. I've been looking on the CrossWire site and the mailing lists but I'm not seeing anything clear, nor is there anything in the Issues mentioning Nestle or the NA28 by name. I would appreciate your feedback. Thanks!

**Comment by mdbergmann** [comment]
Hello, I'm not sure. Does it require a different unlock mechanism than what SWORD provides?

**Reply by MC1171611** [reply]
Maybe I'm just missing how it's supposed to work. Am I the only person to ask about it? I don't know how to use the unlock code/file and maybe I just haven't looked hard enough.

**Reply by mdbergmann** [reply]
Well, I'm not sure the NA28 uses a different mechanism than what is standard for locked modules. If not, then on the left side module tree you can right-click on the locked module (it should show a lock icon) and choose "Unlock module" from the context menu. Then a text window appears where you should enter your unlock code.
After confirming you might have to re-start Eloquent (I'm not sure about this). But that should be it.
